### PR TITLE
✨Source Confluence: Make Connector Compatible with Builder 

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
* Make the connector compatible with the builder

## How
* Import schema to manifest
* Update docs, 
* Completely made compatible with the builder

## User Impact
* No impact, chore for maintainability.

## Can this PR be safely reverted and rolled back?
- [x] YES 💚

### CODE REVIEW

**Comment by btkcodedev** [comment]
Removed $parameter macro
GitHub CI Green

**Comment by btkcodedev** [comment]
Removed $parameter macro :white_check_mark: 
GitHub CI Green :white_check_mark:

**Comment by btkcodedev** [comment]
GitHub CI green, Please continue merge, 
image

File: docs/integrations/sources/confluence.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ The Confluence connector should not run into Confluence API limitations under no
 
 | Version | Date       | Pull Request                                             | Subject                                                                         |
 | :------ | :--------- | :------------------------------------------------------- | :------------------------------------------------------------------------------ |
+| 0.2.4   | 2024-05-30 | [38137](https://github.com/airbytehq/airbyte/pull/38137) | Make connector compatable with the builder                                      |
```
Comment (girarda) [inline docs/integrations/sources/confluence.md:63]:
```suggestion
| 0.2.4   | 2024-05-13 | [38137](https://github.com/airbytehq/airbyte/pull/38137) | Make connector compatible with the builder                                      |
```

**Review by girarda — APPROVED** [review]
approved pending fixing the changelog

**Review by girarda — APPROVED** [review]
approved pending fixing the changelog

**Pull request merged**